source/extensions/filters/http/ext_proc/client_impl.cc

```diff
@@ -32,7 +34,9 @@ ExternalProcessorStreamPtr ExternalProcessorStreamImpl::create(
       std::unique_ptr<ExternalProcessorStreamImpl>(new ExternalProcessorStreamImpl(callbacks));
 
   if (stream->startStream(std::move(client), options)) {
-    stream->stream_->setWatermarkCallbacks(*decoder_watermark_callbacks);
+    if (Runtime::runtimeFeatureEnabled("envoy.reloadable_features.grpc_side_stream_flow_control")) {
```

Comment:
I think we need to latch this to make sure we don't end up wedged if we flip midway through stream lifetime
/wait

Reply:
make sense. Done.

PTAL, thanks!